cmd/sqlcmd/sqlcmd.go

```diff
@@ -118,23 +120,172 @@ func (a SQLCmdArguments) authenticationMethod(hasPassword bool) string {
+
+	rootCmd.Flags().StringVarP(&args.ApplicationIntent, "application-intent", "K", "default", "Declares the application workload type when connecting to a server. The only currently supported value is ReadOnly. If -K is not specified, the sqlcmd utility will not support connectivity to a secondary replica in an Always On availability group")
+	rootCmd.Flags().StringVarP(&args.EncryptConnection, "encrypt-connection", "N", "default", "This switch is used by the client to request an encrypted connection.")
+	rootCmd.Flags().StringVarP(&args.Format, "format", "F", "horiz", "Specifies the formatting for results.")
+	rootCmd.Flags().IntVarP(&args.ErrorsToStderr, "errors-to-stderr", "r", -1, "Controls which error messages are sent to stdout. Messages that have severity level greater than or equal to this level are sent.")
+
+	rootCmd.Flags().IntVar(&args.DriverLoggingLevel, "driver-logging-level", 0, "Level of mssql driver messages to print.")
+	rootCmd.Flags().BoolVarP(&args.ExitOnError, "exit-on-error", "b", false, "Specifies that sqlcmd exits and returns a DOS ERRORLEVEL value when an error occurs.")
+	rootCmd.Flags().IntVarP(&args.ErrorLevel, "error-level", "m", 0, "Controls which error messages are sent to stdout. Messages that have severity level greater than or equal to this level are sent.")
+
+	//Need to decide on short of Header , as "h" is already used in help command in Cobra
+	rootCmd.Flags().IntVarP(&args.Headers, "headers", "h", 0, "Specifies the number of rows to print between the column headings. Use -h-1 to specify that headers not be printed.")
+
+	rootCmd.Flags().BoolVarP(&args.UnicodeOutputFile, "unicode-output-file", "u", false, "Specifies that all output files are encoded with little-endian Unicode")
+	rootCmd.Flags().StringVarP(&args.ColumnSeparator, "column-separator", "s", "", "Specifies the column separator character. Sets the SQLCMDCOLSEP variable.")
+	rootCmd.Flags().BoolVarP(&args.TrimSpaces, "trim-spaces", "W", false, "Remove trailing spaces from a column.")
+	rootCmd.Flags().BoolVarP(&args.MultiSubnetFailover, "multi-subnet-failover", "M", false, "Provided for backward compatibility. Sqlcmd always optimizes detection of the active replica of a SQL Failover Cluster.")
+
+	rootCmd.Flags().StringVarP(&args.Password, "password", "P", "", "Obsolete. The initial passwords must be set using the SQLCMDPASSWORD environment variable or entered at the password prompt.")
+
+	// Using PersistentFlags() for ErrorSeverityLevel due to data type uint8 , which is not supported in Flags()
+	rootCmd.PersistentFlags().Uint8VarP(&args.ErrorSeverityLevel, "error-severity-level", "V", 0, "Controls the severity level that is used to set the ERRORLEVEL variable on exit.")
+	// ScreenWidth is *int type , need to find corresponding Flag Type in PFlag
```

Comment:
>e , need to find corresponding Flag Type in PFlag [](http://example.com/codeflow?start=27&length=49)

i think you'd use the Lookup function to decide if ScreenWidth will be set to a non-null value